crates/iroha_data_model/src/parameter.rs

```diff
@@ -443,6 +443,14 @@ impl Default for SmartContractParameters {
     }
 }
 
+impl Extend<Parameter> for Parameters {
```

Comment:
Probably `impl FromIterator<Parameter> for Parameters` is more symmetric and better